src/util.jl

```diff
@@ -361,11 +361,11 @@ julia> finddelay([1, 2, 3], [0, 0, 1, 2, 3])
 ```
 """
 function finddelay(x::AbstractVector{<: Real}, y::AbstractVector{<: Real})
-    s = xcorr(y, x)
+    s = xcorr(y, x, padmode=:none)
     max_corr = maximum(abs, s)
     max_idxs = findall(x -> abs(x) == max_corr, s)
 
-    center_idx = cld(length(s), 2)
```

Comment:
I don't think this changed with padding

Reply:
The `:longest` to `:none` change requires this. Compare:
```julia
julia> xcorr([0, 0, 1, 2, 3], [1, 2, 3], padmode=:longest)
9-element Array{Int64,1}:
  0
  0
  0
  0
  3
  8
 14
  8
  3
```
Maximum is two indices after the middle => delay of two.
```julia
julia> xcorr([0, 0, 1, 2, 3], [1, 2, 3], padmode=:none)
7-element Array{Int64,1}:
  0
  0
  3
  8
 14
  8
  3
```
Maximum is two indices after the length of the second argument => delay of two.

If you insist, we can change the `padmode` to `:longest` and keep this as is. But I'm more used to the `padmode=:none` behavior.

Reply:
Oh never mind, I get it now.

Reply:
Whoops, I was typing that as you were also t yping. Thanks for the explanation!